kibana-reports/public/components/report_definitions/create/create_report_definition.tsx

```diff
@@ -150,6 +171,50 @@ export function CreateReport(props) {
     timeTo: new Date(),
   };
 
+  const definitionInputValidation = async (metadata, error) => {
+    // check report name
+    // allow a-z, A-Z, 0-9, (), [], ',' - and _ and spaces
+    let regexp = /^[\w\-\s\(\)\[\]\,\_\-+]+$/;
+    if (metadata.report_params.report_name.search(regexp) === -1) {
+      setShowSettingsReportNameError(true);
+      error = true;
+    }
+    // if recurring by interval and input is not a number
+    if (
+      metadata.trigger.trigger_type === 'Schedule' &&
+      metadata.trigger.trigger_params.schedule_type === 'Recurring'
+    ) {
+      let interval = parseInt(
+        metadata.trigger.trigger_params.schedule.interval.period
+      );
+      if (isNaN(interval)) {
+        setShowTriggerIntervalNaNError(true);
+        error = true;
+      }
+    }
+
+    // if cron based and cron input is invalid
+    if (
+      metadata.trigger.trigger_type === 'Schedule' &&
+      metadata.trigger.trigger_params.schedule_type === 'Cron based'
+    ) {
+      if (
+        !isValidCron(metadata.trigger.trigger_params.schedule.cron.expression)
+      ) {
+        setShowCronError(true);
+        error = true;
+      }
+    }
+    // if email delivery and no recipients are listed
+    if (metadata.delivery.delivery_type === 'Channel') {
+      if (metadata.delivery.delivery_params.recipients.length === 0) {
```

Comment:
can you add some basic check for email string?, e.g. it has to be in the format of `xxx@xxx`